datasync/chaindatafetcher/kafka/kafka.go

```diff
@@ -104,7 +104,9 @@ func (k *Kafka) setupTopic(topicName string) error {
 
 	if err := k.CreateTopic(topicName); err != nil {
 		logger.Error("creating a topic is failed", "topicName", topicName, "err", err)
-		return err
+		if kerr, ok := err.(*sarama.TopicError); !ok || kerr.Err != sarama.ErrTopicAlreadyExists {
```

Comment:
Is it okay that we don't handle other error cases?

Reply:
It seems that only `ErrTopicAlreadyExists` can be ignored when creating a new topic [sarama/errors.go](https://github.com/Shopify/sarama/blob/main/errors.go#L129)

Reply:
It seems that only `ErrTopicAlreadyExists` can be ignored when creating a new topic, [sarama/errors.go](https://github.com/Shopify/sarama/blob/main/errors.go#L129)